cli/main.c

```diff
@@ -1434,6 +1434,88 @@ static int fw_info(int argc, char **argv)
 	return 0;
 }
 
+static char * part_to_string(int partition_id) {
+       switch(partition_id){
+
```

Comment:
We use the Linux code style in this project. Please review the documentation for that before submitting:

https://www.kernel.org/doc/html/latest/process/coding-style.html

The first two lines have a ton of style issues:

- Pointer should not have a space after the asterisk
- The open brace for a function should be on it's own line
- There should be a space after "switch"
- There should be a space before the open brace of switch
- Tabs should be used instead of 8 spaces.

Reply:
Fixed the style issues and checked the patch using checkpatch.pl script. 
Commit: 10889e6acb50ac44dce8a53e9cd622d993713aba